mcp/pacemaker.sysconfig

```diff
@@ -1,95 +1,102 @@
+# Log all messages from a comma-separated list of functions.
 # PCMK_trace_functions=function1,function2,function3
 
-# Log all messages from a comma-separated list of files (no path)
-# Supports wildcards eg. PCMK_trace_files=prefix*.c
+# Log all messages from a comma-separated list of files (no path).
+# Supports wildcards eg. PCMK_trace_files=prefix*.c.
 # PCMK_trace_files=file.c,other.h
 
-# Log all messages matching comma-separated list of formats
+# Log all messages matching comma-separated list of formats.
 # PCMK_trace_formats="Sent delete %d"
 
-# Log all messages from a comma-separated list of tags
+# Log all messages from a comma-separated list of tags.
 # PCMK_trace_tags=tag1,tag2
 
 # Dump the blackbox whenever the message at function and line is printed
-# eg. PCMK_trace_blackbox=te_graph_trigger:223,unpack_clone:81
+# eg. PCMK_trace_blackbox=te_graph_trigger:223,unpack_clone:81.
 # PCMK_trace_blackbox=fn:line,fn2:line2,...
 
-# Enable blackbox logging globally or per-subsystem
-# The blackbox contains a rolling buffer of all logs (including info+debug+trace)
-# and is written after a crash, assertion failure and/or when SIGTRAP is received
+# Enable blackbox logging globally or per-subsystem.
+# The blackbox contains a rolling buffer of all logs (incl. info+debug+trace)
+# and is written after a crash, assertion failure and/or when SIGTRAP is
+# received.
 #
-# The blackbox recorder can also be enabled for Pacemaker daemons at runtime by 
-# sending SIGUSR1 (or SIGTRAP), and disabled by sending SIGUSR2
+# The blackbox recorder can also be enabled for Pacemaker daemons at runtime
+# by sending SIGUSR1 (or SIGTRAP), and disabled by sending SIGUSR2.
 #
 # Multiple subsystems may me listed separated by commas
-# eg. PCMK_blackbox=crmd,pengine
+# eg. PCMK_blackbox=crmd,pengine.
 # PCMK_blackbox=yes|no|crmd|pengine|cib|stonith-ng|attrd|pacemakerd
 
 #==#==# Advanced use only
 
 # Enable this for compatibility with older corosync (prior to 2.0)
-# based clusters which used the nodes uname as its uuid also
+# based clusters which used the nodes uname as its uuid also.
 # PCMK_uname_is_uuid=no
 
-# Specify an alternate location for RNG schemas and XSL transforms
-# Mostly only useful for developer testing
+# Specify an alternate location for RNG schemas and XSL transforms.
+# Mostly only useful for developer testing.
 # PCMK_schema_directory=/some/path
 
-# Enable this for rebooting this machine at the time of process (subsystem) failure
+# Enable this for rebooting this machine at the time of process (subsystem)
+# failure.
 # PCMK_fail_fast=no
 
-# Set action at the time of the system reboot by fail_fast
-# When set 'crash' , the kernel does panic
-# When you want to get kdump , please set 'crash'
-# The default action is 'reboot'
+# Set action at the time of the system reboot by fail_fast.
+# When set 'crash' , the kernel does panic.
+# When you want to get kdump , please set 'crash'.
+# The default action is 'reboot'.
```

Comment:
Maybe: 'Pacemaker will panic its host under certain conditions. If this is set to "crash", Pacemaker will trigger a kernel crash (which is useful if you want a kernel dump to investigate). For any other value, Pacemaker will trigger a host reboot. The default is unset.'